src/xbot/common/subsystems/BasePoseSubsystem.java

```diff
@@ -47,8 +47,7 @@
     
     private BooleanProperty rioRotated;
     
-    @Inject
-    public PoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
+    public BasePoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
```

Comment:
I think `TelemetrySource` just might be a bad name for what's going on here, there's a class of things we want to run every robot tick regardless of whats going on in terms of enabled/disabled (one of which is updating telemetry on the dashboard but another is this kind of sensor integration). Open to better names for `TelemetrySource` given the broader set of scenarios it's really meant for.